#!/usr/bin/env python
PACKAGE = "modelos_cinematicos"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("LOWER_elbow", bool_t , 0, "Orientaciòn deseada de codo",False)
gen.add("Grafica_Python", bool_t , 0, "Dibuja la gràfica de Espacio de trabajo en Python",False)
gen.add("Workspace", bool_t , 0, "Dibujar espacio de trabajo",False)
gen.add("Borrar_workspace", bool_t , 0, "Borrar espacio de trabajo",False)
gen.add("Cuadrado", bool_t , 0, "Trayectoria Cuadrada",False)
gen.add("Triangulo", bool_t , 0, "Trayectoria Triangular",False)
gen.add("Circulo", bool_t , 0, "Trayectoria Circular",False)
gen.add("mov_inicial", bool_t , 0, "Marcar para enviar a la posición inicial",False)
gen.add("Simulacion", bool_t , 0, "Realizar trayectoria unicamente con simulacion",False)
gen.add("Dynamixel", bool_t , 0, "Realizar trayectoria con robot real y simulado",False)
gen.add("Solo_Dynamixel", bool_t , 0, "Realizar trayectoria unicamente con robot real",False)
gen.add("Ready", bool_t , 0, "Marcar para iniciar trazo de trayectoria",False)
exit(gen.generate(PACKAGE, "trayectorie_2dof", "trayectoria_3g"))
